src/swupd_lib/delta.c

```diff
@@ -168,20 +167,15 @@ void apply_deltas(struct manifest *current_manifest)
 
 			if (!compute_hash_from_file(filename, hash) || !hash_equal(file->hash, hash)) {
 				FREE(filename);
-				bad_on_sys = true;
+				warn("File '%s' is missing or corrupted - consider running 'swupd repair'\n", file->filename);
```

Comment:
In other parts of the code when we show this type of warning we do it in two steps:
```
warn("My warning\n");
info("My suggestion\n");
```
You can see an example of this here: https://github.com/clearlinux/swupd-client/blob/master/src/cmds/verify.c#L673
Also I would use "command" instead of 'command' as in the example above to keep it consistent.

Reply:
cool, I followed your suggestion. Can you take another look

Reply:
Looks good, the only thing you didn't do was using "" instead of `` like this:
`info("Consider running /"swupd repair/" to fix the issue\n");`